react-notes/src/layouts/MainLayout/components/Sidebar/Sidebar.js

```diff
@@ -4,17 +4,24 @@ import { Link } from 'react-router-dom';
 import styles from './Sidebar.module.css';
 import { ArticleLink } from './components';
 import { ArticlesContext } from '../../../../pages/article/context';
+import classNames from 'classnames';
 
 export const Sidebar = (props) => {
     const { articles } = useContext(ArticlesContext);
 
     return (
-        <div {...props} className={styles.navbar}>
+        <div {...props} className={styles.sidebar}>
             <Link to="article/create" className={styles.addBtn}>
                 Add article
             </Link>
 
-            <h3 className={styles.title}>Articles: </h3>
+            <h3
+                className={classNames(styles.title, {
+                    [styles.noArticles]: !articles.length,
```

Comment:
```suggestion
                    [styles.noArticles]: articles.length === 0,
```

Reply:
Is it for more clearly understand of code?

Reply:
Yes and no

Generally, negation operator (`!`) is used to determine **existence** of something, e.g. if articles *array* exists then we push new values there; or if userObject exists then get username from it.

Here it is rather a question if articles length IS NOT 0, or if articles array IS NOT empty. For this I'd rather use `=== 0`.

... or I would recommend taking a look at `lodash` library, that has an amazing function called `isEmpty`, which will return boolean value on any "falsy" value - `null, undefined, '', [], 0` etc.

Reply:
Just to make sure - `lodash` is good not only because of `isEmpty`, but in this case this function would be useful